.github/workflows/build.yml

```diff
@@ -262,16 +262,16 @@ jobs:
           git diff --name-only --exit-code && echo "✅ Generated proto matches committed proto" || (echo "❌ Generated proto differs from committed proto, run \`make proto -B\` and commit the result" >&2 && exit 1)
           make test
 
-  # Verify go sdk unit tests
+  # Verify go sdk unit/fuzz tests
   sdk_vaa:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
           go-version: "1.20.4"
-      - run: cd sdk/vaa && go test
-
+      - run: cd sdk/vaa && go test && go test -v -fuzz FuzzCalculateQuorum -run FuzzCalculateQuorum -fuzztime 15s
```

Comment:
I'm currently adding fuzz tests explicitly based on their need to be added, but maybe in if we get a enough of these can add a script harness and then pick them all up.  I think the main consideration is how much time penalty these may or may not add to the run, which is why this is timebound to 15s.

Reply:
The supplied harness however, allows anyone with more interest, compute and willingness to guide the fuzzer to quickly dive in and make some progress there.